web/components/Nav/Nav.module.css

```diff
@@ -1,22 +1,138 @@
 .nav {
-  display: flex;
-  justify-content: center;
-  background: #eee;
-  padding: 1rem;
-  margin: 1rem;
+  box-shadow: 0 6px 0 rgba(0, 0, 0, 0.1);
 }
 
-.nav__items {
+.menuContents {
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
-  flex: 1;
-  flex-wrap: wrap;
+  align-items: center;
+  padding: 20px 0;
+}
+
+.logo {
+  width: auto;
+  height: 48px;
+}
+
+.items {
   list-style: none;
+  display: flex;
+}
+
+.menuButtonWrapper,
+.ticketItem,
+.closeButton {
+  display: none;
+}
+
+.link {
+  color: var(--color-brand-black);
+  display: block;
+  margin: 0 10px;
+  text-decoration: none;
+  font: var(--font-ui-sm-medium);
+  letter-spacing: var(--letter-spacing-ui-sm);
+}
+
+.link:hover,
+.link:focus {
+  text-decoration: underline;
 }
 
-@media screen and (max-width: 480px) {
-  .nav__items {
-    max-width: 200px;
+@media not all and (--medium-up) {
```

Comment:
Nit: this reads very strange to me. Might be because I'm not familiar with the spec ([I just read up on this](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#inverting_a_querys_meaning)), but can we create a `--medium-down` variable instead, so that we can simply write `@media (--medium-down)`? (If medium-down is what we want, that is.)